src/pages/index.js

```diff
@@ -137,16 +137,17 @@ function createCard(data) {
 
 
 function handleProfileEditSubmit(data) {
+editProfilePopup.renderLoading(true);
+
   api.updateProfile(data.name, data.profession)
     .then((data) => {
       userInfo.setUserInfo(data)
-      editProfilePopup.renderLoading(true)
+      editProfilePopup.close();
     })
-    .finally(editProfilePopup.renderLoading(false))
     .catch((err) => {
       console.log(console.error);
```

Comment:
`console.log(console.error);` should be `console.error(err);`. Please, pay attantion the `console.error` is a method for logging errors